### PR TITLE
Update BSI to point to the d2l-navigation-band component in d2l-navigation

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -238,7 +257,15 @@
       "Sortable": "8794188794",
       "stickyfill": "2.0.5",
       "susy": "2.2.14",
+      "vui-breadcrumbs": "1a0c1af674",
```
Comment (svanherk) [inline bower.json]: Looks like something went wrong here, these were all recently removed from bsi

File: bower.json [hunk 2]
```diff
@@ -77,6 +78,14 @@
...
     "stickyfill": "https://github.com/wilddeer/stickyfill.git#dfa336fb0723cae19e65eeb1986394ad565f7e6b",
     "susy": "https://github.com/ericam/susy.git#68b421259112eb882d2d1103738bee3b703e6bf3",
+    "vui-breadcrumbs": "https://github.com/Brightspace/valence-ui-breadcrumbs.git#1a0c1af6745a73b601597eb0d965df9c329bbc12",
+    "vui-dropdown": "https://github.com/Brightspace/valence-ui-dropdown.git#5c50e4d8ae2d19b33514c477c20812f2264837f0",
+    "vui-field": "https://github.com/Brightspace/valence-ui-field.git#613d3cc4fb370ac6ed020fecda4c64c089ed4610",
+    "vui-focus": "https://github.com/Brightspace/valence-ui-focus.git#d092b7aa3c9eff70cbbbfc8567317f31fee4e563",
+    "vui-grid-system": "https://github.com/Brightspace/valence-ui-grid-system.git#0384e87cd068617419d42b3c8a8b6cb773fc507d",
+    "vui-input": "https://github.com/Brightspace/valence-ui-input.git#52a259d79389632bae31830c32d7303a5db14345",
+    "vui-list": "https://github.com/Brightspace/valence-ui-list.git#95245dcfc833c1eae49f3d840ba12b6c2674a55a",
+    "vui-validation": "https://github.com/Brightspace/valence-ui-validation.git#757daf55e9fad9aacf31dd34ace60c5015ed3225",
```
Comment (dlockhart) [inline bower.json]: These shouldn't be coming back... make sure you empty your `bower_components` before `bower install`ing.

Comment (jeroach) [reply bower.json]: Sounds good - will fix.

Comment (jeroach) [reply bower.json]: https://github.com/Brightspace/brightspace-integration#bower-locking

**Pull request merged**